hoot-core/src/main/cpp/hoot/core/ops/CookieCutterOp.cpp

```diff
@@ -57,43 +61,61 @@ void CookieCutterOp::setConfiguration(const Settings& conf)
 
 void CookieCutterOp::apply(std::shared_ptr<OsmMap>& map)
 {
```

Comment:
I think there are some projection issues going on here. I see them with some datasets and not others. Ultimately I ended up not using this class in the replacement changeset changes, b/c it wouldn't do everything I needed. I commented out what thought were fixes by reprojecting after they made some of the test output look strange.